legate/jupyter/_magic_cmd.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# mypy: ignore-errors
+import json
+import os
+import sys
+
+from IPython.core.magic import Magics, line_magic, magics_class
+from jupyter_client.kernelspec import KernelSpecManager, NoSuchKernel
+
+cmd_dict = {
+    "cpus": "Number of CPUs to use per rank",
+    "gpus": "Number of GPUs to use per rank",
+    "openmp": "Number of OpenMP groups to use per rank",
```

Comment:
The legate driver (for better or worse) uses `--omps` for this:
```
  --omps OPENMP         Number of OpenMP groups to use per rank
```
Would it be better to match the keys here to the legate driver command line options exactly?

Reply:
Thank you, fixed in the latest commit.